pkg/manager/member/utils.go

```diff
@@ -532,3 +540,30 @@ func GetPVCSelectorForPod(controller runtime.Object, memberType v1alpha1.MemberT
 	}
 	return l.Selector()
 }
+
+// TiKVLessThanV50 checks whether the `image` is less than v5.0.0
+func TiKVLessThanV50(image string) bool {
+	_, version := parseImage(image)
+	v, err := semver.NewVersion(version)
+	if err != nil {
+		klog.Errorf("Parse version %s failure, error: %v", version, err)
+		return false
```

Comment:
what's the behavior if omit the tag (eg for `latest`)?

Reply:
It will return false and then use the new path of the log file same with a version of TiKV is more than v5.0.0.